Automattic/wp-calypso
# Assembler: Update animation in the Pages screen

## Proposed Changes

This PR adds animations when the user clicks on a page preview. See the design reference in pbxlJb-4EO-p2.

<ins>Note</ins>: Due to the complexity of keeping the expanded page preview centered, this PR will close the expanded page preview on browser window resize. We can discuss if it's worth supporting windows resize later on.

Before:

https://github.com/Automattic/wp-calypso/assets/797888/76cbbc97-6500-4119-8558-a71b028534a6

After:

https://github.com/Automattic/wp-calypso/assets/797888/18f35803-77e4-4b21-a4e8-dfb1c764f9af


## Testing Instructions



* Head to the Assembler.
* Select any patterns and colors.
* Once in the Pages screen, add some pages.
* Click on the page previews and ensure that they feel great.

## Pre-merge Checklist
